client/js/lounge.js

```diff
@@ -146,6 +146,7 @@ $(function() {
 				})
 			);
 			var channels = $.map(data.networks, function(n) {
+				sidebar.find("#network-" + n.id).data("options", n.serverOptions);
```

Comment:
@xPaw, I moved this myself to the template in 9dd980d. What do you think? I can remove if you dislike.